src/styles/blocks/_global.scss

```diff
@@ -0,0 +1,15 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+  padding-top: 50px;
```

Comment:
Remove the padding-top property, as it's not needed when using align-items: center and justify-content: center to center the calendar.

Reply:
Done! Need push and deploy?